src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import AlertBox, {
+  ALERT_TYPE,
+} from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+const ViewPaymentHistoryCTA = props => {
+  let alertType;
+  let headline;
+  let alertContent;
+  if (props.includedInFlipper === undefined) {
+    return <LoadingIndicator message="Loading..." />;
+  } else if (props.includedInFlipper === false) {
+    alertContent = (
+      <>
+        <p>
+          To use this feature, you'll need a Premium <strong>DS Login</strong>{' '}
+          account. Your My HealtheVet or ID.me credentials won't work on the
+          eBenefits website. Go to eBenefits to sign in, register, or upgrade
+          your <strong>DS Login</strong> account to Premium.
+        </p>
+        <a
```

Comment:
You can use the `EbenefitsLink` Component from platform similar to [our 686 cta](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/view-modify-dependent/686-cta/containers/Form686CTA.jsx#L17) It has built in analytics firing.

Reply:
Added